xcode/Subconscious/Shared/Parsers/Header.swift

```diff
@@ -14,6 +14,7 @@ enum HeaderName: String {
     case created = "Created"
     case modified = "Modified"
     case fileExtension = "File-Extension"
+    case color = "Color"
```

Comment:
IMO the header name should be `Theme-Color` and the case name `themeColor` 

Prior art: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color

Reply:
I don't think the header should store constant names. Instead, we should consider storing hex values.

Should we include support for color variants?

```
Theme-Color: #abc123
Theme-Color-Dark: #abc-123
```

Reply:
Updated the header name, see https://github.com/subconsciousnetwork/subconscious/pull/1064#issuecomment-1884141408 RE: storing hex.